Natives_In_Tech_Bylaws.md

```diff
@@ -94,7 +94,7 @@ An Advisory Council may be created whose members shall be elected by the members
 
 **Section 13. Parliamentary Procedure**
 
-Any question concerning parliamentary procedure at meetings shall be determined by the President by reference to Robert's Rules of Order.
+Any question concerning parliamentary procedure shall be determined by the President by reference the organization's [rules of order](Git_Rules_of_Order.md).
```

Comment:
*by referencing

Reply:
Ah I deleted `to`.

Reply:
@erinishimoticha Did you push up your changes?

Reply:
Whoops, pushed now.